naga/tests/in/interface.param.ron

```diff
@@ -1,3 +1,4 @@
+//TODO: GLSL https://github.com/gfx-rs/naga/issues/874
```

Comment:
nitpick: We should move this to be above `targets`, rather than at the top.

Reply:
Moved it by targets, and moved targets to the top.